route_planning_msgs/msg/RouteElement.msg

```diff
@@ -0,0 +1,11 @@
+LaneElement[] lane_elements             # cross sections of lanes in driving direction (from left to right)
+
+geometry_msgs/Point left_boundary       # left boundary of drivable space
+geometry_msgs/Point right_boundary      # right boundary of drivable space
```

Comment:
if RouteElement is not enriched, these are 0? Do we need a "is_valid" flag?

Reply:
We could add a `is_enriched` flag to the RouteElement, or e.g. `has_right / has_left_boundary` as implemented in the LaneElement.

Reply:
-> add `is_enriched` to RouteElement, remove `has_left_boundary` etc.